examples/gno.land/p/demo/audio/biquad/biquad.gno

```diff
@@ -0,0 +1,57 @@
+package biquad
+
+import "math"
+
+type Filter struct {
+	a1, a2, b0, b1, b2     float64
+	x1_f, x2_f, y1_f, y2_f float64
+}
+
+func New(fc float64, fs float64, q float64, db float64, filter_type string) (f *Filter) {
```

Comment:
```suggestion
func New(fc float64, fs float64, q float64, db float64, filterType string) *Filter {
```

(we don't used the named return parameter)

also add some doc, especially on what each parameter is?

Reply:
Thanks, didn't know about the return convention, will follow here on out. Update is here: 6939ecd7